src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -65,10 +65,14 @@ class Container implements ResettableContainerInterface
 
     protected $services = array();
     protected $methodMap = array();
-    protected $privates = array();
     protected $aliases = array();
     protected $loading = array();
 
+    /**
+     * @deprecated since 3.2, to be removed in 4.0
+     */
+    protected $privates = array();
```

Comment:
What the annotation says is that one should not rely on this property since it will be removed in 4.0. Of course, when doing so, we will also make the required changes to *not* bring back the original issue.